README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<img width=75% src="/gridstatus-header.png" alt="gridstatus logo" />
+<img width=75% src="https://github.com/gridstatus/gridstatus/blob/main/gridstatus-header.png" alt="gridstatus logo" />
```

Comment:
I changed this so hopefully the image will appear on PyPI.

<img width="878" alt="Screenshot 2024-10-03 at 11 17 15" src="https://github.com/user-attachments/assets/cf5066b0-fe05-423e-baf7-7792b28e08fc">